tsconfig.json

```diff
@@ -1,10 +1,12 @@
 {
     "compilerOptions": {
         "target": "ES2018",
-        "module": "ESNext",
+        "module": "CommonJS",
```

Comment:
Why?

Reply:
ah crap, thats not supposed to be there. got an error about importing `webpack.DefinePlugin` in the config file and was testing things trying to get rid of it... good catch :)